src/scss/layout/_cta.scss

```diff
@@ -0,0 +1,26 @@
+@use '../../../node_modules/accoutrement/sass/tools';
+
+[data-layout='cta'] {
+  --media-size: calc(4em + 2vw);
+  background: var(--callout-fade);
+  clip-path: polygon(
+    0 0,
+    calc(50% - var(--double-gutter)) 0,
+    50% var(--double-gutter),
+    calc(50% + var(--double-gutter)) 0,
+    100% 0,
+    100% calc(100% - var(--double-gutter)),
+    calc(50% + var(--double-gutter)) calc(100% - var(--double-gutter)),
+    50% 100%,
+    calc(50% - var(--double-gutter)) calc(100% - var(--double-gutter)),
+    0 calc(100% - var(--double-gutter))
+  );
+  display: grid;
+  font-size: var(--small);
+  grid-area: cta;
+  grid-template-columns: minmax(min-content, var(--small-page));
+  justify-content: center;
+  margin: 0 calc(var(--body-inline) * -1);
+  padding: var(--spacer) var(--body-inline) calc(var(--spacer) + var(--gutter));
+  text-shadow: 0 1px 0 var(--shadow);
```

Comment:
hmmm, I only saw the text shadow when I selected text. Is that expected?

Reply:
I removed it already in another branch…